README.md

```diff
@@ -33,12 +33,18 @@ helm install --name=kafka-minion kafka-minion/kafka-minion
 | `kafka.brokers` | Comma delimited list of brokers to connect to | (none) |
 | `kafka.sasl.enabled` | Bool to enable/disable SASL authentication (only SASL_PLAINTEXT is supported) | `false` |
 | `kafka.sasl.useHandshake` | Whether or not to send the Kafka SASL handshake first | `true` |
-| `kafka.sasl.credentials.existingSecret` | Secretname of an existing secret which contains SASL credentials | (none) |
+| `kafka.sasl.existingSecret` | Secretname of an existing secret which contains SASL credentials | (none) |
+| `kafka.sasl.existing.username` | Keyname of an existing key on an existing secret which contains SASL username | `username` |
```

Comment:
`kafka.sasl.existing.usernameKey` is more accurate I think? Accordingly for the other property names

Reply:
I went back and forth with that naming scheme and what I currently have. I am indifferent. I will update the PR to suffix these new values with `Key`.